docs/source/en/using-diffusers/contribute_pipeline.mdx

```diff
@@ -85,71 +91,72 @@ pipeline = UnetSchedulerOneForwardPipeline(unet=unet, scheduler=scheduler)
 output = pipeline()
 ```
 
-But what's even better is that you can load pre-existing weights into the pipeline if they match exactly your pipeline structure. This is e.g. the case for [https://huggingface.co/google/ddpm-cifar10-32](https://huggingface.co/google/ddpm-cifar10-32) so that we can do the following:
+But what's even better is you can load pre-existing weights into the pipeline if the pipeline structure is identical. For example, you can load the [`google/ddpm-cifar10-32`](https://huggingface.co/google/ddpm-cifar10-32) weights into the one-step pipeline:
 
 ```python
 pipeline = UnetSchedulerOneForwardPipeline.from_pretrained("google/ddpm-cifar10-32")
 
 output = pipeline()
 ```
 
-We want to share this amazing pipeline with the community, so we would open a PR request to add the following code under `one_step_unet.py` to [https://github.com/huggingface/diffusers/tree/main/examples/community](https://github.com/huggingface/diffusers/tree/main/examples/community) .
-
-```python
-from diffusers import DiffusionPipeline
-import torch
-
+## Share your pipeline
 
-class UnetSchedulerOneForwardPipeline(DiffusionPipeline):
-    def __init__(self, unet, scheduler):
-        super().__init__()
+Open a Pull Request on the 🧨 Diffusers [repository](https://github.com/huggingface/diffusers) to add your awesome pipeline in `one_step_unet.py` to the [examples/community](https://github.com/huggingface/diffusers/tree/main/examples/community) subfolder.
 
-        self.register_modules(unet=unet, scheduler=scheduler)
+Once it is merged, anyone with `diffusers >= 0.4.0` installed can use this pipeline magically 🪄 by specifying it in the `custom_pipeline` argument:
 
-    def __call__(self):
-        image = torch.randn(
-            (1, self.unet.in_channels, self.unet.sample_size, self.unet.sample_size),
-        )
-        timestep = 1
-
-        model_output = self.unet(image, timestep).sample
-        scheduler_output = self.scheduler.step(model_output, timestep, image).prev_sample
+```python
+from diffusers import DiffusionPipeline
 
-        return scheduler_output
+pipe = DiffusionPipeline.from_pretrained("google/ddpm-cifar10-32", custom_pipeline="one_step_unet")
+pipe()
 ```
 
-Our amazing pipeline got merged here: [#840](https://github.com/huggingface/diffusers/pull/840).
-Now everybody that has `diffusers >= 0.4.0` installed can use our pipeline magically 🪄 as follows:
+Another way to share your community pipeline is to upload the `one_step_unet.py` file directly to your preferred [model repository](https://huggingface.co/docs/hub/models-uploading) on the Hub. Instead of specifying the `one_step_unet.py` file, pass the model repository id to the `custom_pipeline` argument:
```

Comment:
What's the difference between having the pipeline on GitHub vs Hugging Face Hub? It seems like the GitHub ones are "official" and the Hub ones aren't. The only difference seems to be the sharing workflow (kind of like the [legacy datasets](https://huggingface.co/docs/datasets/share#datasets-on-github-legacy) from the Datasets library), so I think it'd be nice to clarify for the user here the pros and cons of each sharing workflow.

cc @patrickvonplaten @yiyixuxu @sayakpaul @pcuenca

Reply:
I think the only difference is that GitHub ones are reviewed and added to the repo, but users can still use others from the Hub without going through this process.

Reply:
Is there a recommended way to share the pipeline? I think this can potentially be confusing for users, for example, are the Hub community pipelines not as "official" or "legit" as the GitHub community ones? I made a table comparing the two:

|                | GitHub community pipeline                                             | HF Hub community pipeline                                       |
|----------------|-----------------------------------------------------------------------|-----------------------------------------------------------------|
| visibility     | included in official Diffusers examples                               | relies on your own usage and promotion                          |
| review process | open a Pull Request on GitHub and get it merged by the Diffusers team | upload directly to a Hub repository without any review (faster) |
| usability      | same                                                                  | same                                                            |

Reply:
@stevhliu I think you table is great and that's exactly the trade-off! 
Would be super nice to include it